src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -340,6 +341,7 @@ export class CourseUpdateComponent implements OnInit {
         }
 
         this.router.navigate(['course-management', updatedCourse?.id?.toString()]);
+        scrollToTopOfPage();
```

Comment:
Noticed it when creating a new course, if you scroll down to a course setting and save you will be forwarded to the overview to the course but not scrolled to the top (as we usually do, e.g. for the configuration of programming exercises)